src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -742,6 +742,7 @@ private function registerWorkflowConfiguration(array $config, ContainerBuilder $
                 }
             }
             $metadataStoreDefinition->replaceArgument(2, $transitionsMetadataDefinition);
+            $container->setDefinition(sprintf('%s.metadata_store', $workflowId), $metadataStoreDefinition);
```

Comment:
I finally understood what you meant ^^

Is it possible to expose it as type + named parameter ?